### ISSUE TITLE
Documentation: File doc/users/whats_new/README does not exist

### ISSUE THREAD

**Issue by EJEP** [matplotlib/matplotlib#9200]
In line 214 of `doc/devel/contributing.rst` states that the file `doc/users/whats_new/README` has more information about adding an entry to the `whats_new` section. This file does not exist. The directory `doc/users/whats_new` does not exist either.

The directory `doc/users/next_whats_new` does exist and contains a `README` file.

Is this the file (and directory) `doc/devel/contributing.rst` is referring to?

**Comment by tacaswell** [comment]
Yes, that is an error in the docs (the folder got renamed as part of auto-including the rst files in to the top level whats_new with a glob).
